packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md

```diff
@@ -215,20 +285,40 @@ This status indicates that the user needs to provide additional authenticator in
 
 You need to build a form to capture the user's phone number as well as a subsequent form for the user to select their phone verification method (either SMS or voice).
 
-<div class="half">
+<div class="half wireframe-border">
 
-![Displays the Java SDK's enroll phone number authenticator form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-num-java.png)
+![Phone number entry form](/img/oie-embedded-sdk/wireframes/auth-enter-phone-number-form-g2r32.png)
 
 </div>
 
+<!--
+
+Source image:
+
+https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%8C%9F-Updated-Diagrams-for-Dev-Docs?node-id=2393%3A2128#233281241
+
+Group 2, row 32
+
+-->
+
 > **Note:** The Java SDK requires the following phone number format: `{+}{country-code}{area-code}{number}`. For example, `+15556667777`.
 
-<div class="half">
+<div class="half wireframe-border">
 
-![Displays the Java SDK's phone factor (SMS or voice) form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-mode-java.png)
+![Phone verification factor selection form](/img/oie-embedded-sdk/wireframes/choose-message-type-verification-code-g2r33.png)
```

Comment:
As noted earlier, the title should be "Choose a message type" rather than "Choose your message type" to be consistent across all wireframes

Reply:
Agreed. Replaced with updated screenshot in next commit.